<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>THE BEST PROJECT OF ALL TIMES AND PEOPLES</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCalendarWidget" name="mainCalendar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>831</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="dateData" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>620</y>
      <width>831</width>
      <height>271</height>
     </rect>
    </property>
    <widget class="QLabel" name="curDate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Script MT Bold</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>CurrentDate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>0</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить запись</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>0</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Редактировать запись</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delButton">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>0</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить запись</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="entryTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>841</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
